### PR TITLE
Bolton

### PR DESCRIPTION
**Pull request by WILHEMS**
Updates

### CODE REVIEW

File: src/main/java/com/develhope/imsdevelhope/app/persistence/entities/ItemLocation.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package com.develhope.imsdevelhope.app.persistence.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "ITEM_LOCATION")
```
Comment (develhope-team1) [inline src/main/java/com/develhope/imsdevelhope/app/persistence/entities/ItemLocation.java:8]: tamuu.

File: src/main/java/com/develhope/imsdevelhope/app/persistence/entities/Items.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
+package com.develhope.imsdevelhope.app.persistence.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "ITEMS")
```
Comment (develhope-team1) [inline src/main/java/com/develhope/imsdevelhope/app/persistence/entities/Items.java:9]: good

File: src/main/java/com/develhope/imsdevelhope/app/persistence/entities/Purchases.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package com.develhope.imsdevelhope.app.persistence.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "PURCHASES")
```
Comment (develhope-team1) [inline src/main/java/com/develhope/imsdevelhope/app/persistence/entities/Purchases.java:9]: fity

File: src/main/java/com/develhope/imsdevelhope/app/persistence/entities/Items.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+public class Items {
+    public Integer itemId;
+    public String itemName;
+    public boolean itemAvailable;
+    public Integer itemStock;
+    @Id
+    private Long id;
+
+
+    public Items(Integer itemId, String itemName, boolean itemAvailable, Integer itemStock) {
```
Comment (develhope-team1) [inline src/main/java/com/develhope/imsdevelhope/app/persistence/entities/Items.java]: it would be better if you use a no args constructor.

**Review by develhope-team1 — APPROVED** [review]

File: src/main/java/com/develhope/imsdevelhope/app/persistence/entities/Items.java (hunk 4 above)
Comment (WILHEMS) [reply src/main/java/com/develhope/imsdevelhope/app/persistence/entities/Items.java]: There's a noargs  constructor. Or what do you mean by noargs constructor.

**Pull request closed without merging**